src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -59,6 +65,14 @@ public FunctionsSyncManager(IConfiguration configuration, IHostIdProvider hostId
             _environment = environment;
         }
```

Comment:
nit -- can we change this to take an `ILogger<FunctionsSyncManager>` in the ctor?

Reply:
If we change the logger category like this, it'll make detectors easier to run as they can filter on it. @finvamp1 FYI.

Reply:
done